### PR TITLE
fix/2509-submit-work-modal-small-copy-update

### PR DESCRIPTION
**Pull request by talentdev0807**
#2509

### CODE REVIEW

File: packages/ui/src/services/i18n/dict/en/bounty.json [hunk 1]
```diff
@@ -377,4 +378,4 @@
     "title": "You are slashed",
     "description": "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis"
```
Comment (singulart) [inline packages/ui/src/services/i18n/dict/en/bounty.json:379]: It's good to also replace this text with a meaningful one. @dmtrjsg wdyt?

**Comment by singulart** [comment]
@thesan please re-trigger the CI

**Review by singulart — APPROVED** [review]

**Review by thesan — APPROVED** [review]
Thank you for the PR. And for the extra effort on the translation :open_hands:

File: packages/ui/src/services/i18n/dict/en/bounty.json (hunk 1 above)
Comment (thesan) [reply packages/ui/src/services/i18n/dict/en/bounty.json:379]: Yes but it's done in #3080

**Pull request merged**